Net/src/TCPServer.cpp

```diff
@@ -150,24 +151,30 @@ void TCPServer::run()
 				}
 				catch (Poco::Exception& exc)
 				{
-					ErrorHandler::handle(exc);
+					if (!_stopped)
```

Comment:
This if is always true: _stopped is true at this point.

Reply:
> This if is always `false`: `_stopped` is `true` at this point.

`acceptConnection()` can throw for many reasons other than `_socket.close()` being called on `stop()`